modules/integration/tests-platform/tests-clustering/src/test/java/org/wso2/mb/platform/tests/clustering/slot/SlotDeliveringTestCase.java

```diff
@@ -0,0 +1,158 @@
+import org.wso2.mb.integration.common.clients.operations.utils.ExchangeType;
+import org.wso2.mb.platform.common.utils.MBPlatformBaseTest;
+import org.xml.sax.SAXException;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.rmi.RemoteException;
+
+import static org.testng.Assert.assertTrue;
+
+public class SlotDeliveringTestCase extends MBPlatformBaseTest {
+
+	/**
+	 * Prepare environment for test
+	 * @throws LoginAuthenticationExceptionException
+	 * @throws IOException
+	 * @throws XPathExpressionException
+	 * @throws URISyntaxException
+	 * @throws SAXException
+	 * @throws XMLStreamException
+	 */
+	@BeforeClass(alwaysRun = true)
+	public void init()
+			throws LoginAuthenticationExceptionException, IOException, XPathExpressionException,
+			       URISyntaxException, SAXException, XMLStreamException, AndesAdminServiceBrokerManagerAdminException {
+		super.initCluster(TestUserMode.SUPER_TENANT_ADMIN);
+		super.initAndesAdminClients();
+
+		String randomInstanceKey = getRandomMBInstance();
+
+		AndesAdminClient tempAndesAdminClient = getAndesAdminClientWithKey(randomInstanceKey);
+
+		if (tempAndesAdminClient.getQueueByName("clusterSingleQueue1") == null) {
+			tempAndesAdminClient.createQueue("clusterSingleQueue1");
+		}
+
+		if (tempAndesAdminClient.getQueueByName("clusterSingleQueue2") == null) {
+			tempAndesAdminClient.createQueue("clusterSingleQueue2");
+		}
+
+		if (tempAndesAdminClient.getQueueByName("clusterSingleQueue3") == null) {
+			tempAndesAdminClient.createQueue("clusterSingleQueue3");
+		}
+
+	}
+
+	/**
+	 *
+	 * @throws Exception
+	 */
+	@Test(groups = "wso2.mb", description = "Single queue Single node send-receive test case")
+	public void testSingleQueueSingleNodeSendReceive() throws Exception {
+
+		long sendCount = 1000L;
+		long expectedCount = 1000L;
+
+
+		String randomInstanceKey = getRandomMBInstance();
+
+		AutomationContext tempContext = getAutomationContextWithKey(randomInstanceKey);
+
+		AndesJMSConsumerClientConfiguration consumerConfig = new AndesJMSConsumerClientConfiguration(tempContext.getInstance().getHosts().get("default"),
+		                                                                                             Integer.parseInt(tempContext.getInstance().getPorts().get("amqp")),
+		                                                                                             ExchangeType.QUEUE, "clusterSingleQueue1");
+		consumerConfig.setMaximumMessagesToReceived(expectedCount);
+		consumerConfig.setPrintsPerMessageCount(expectedCount / 10L);
+
+
+		randomInstanceKey = getRandomMBInstance();
+		Queue queue = getAndesAdminClientWithKey(randomInstanceKey).getQueueByName("clusterSingleQueue1");
+
+		assertTrue(queue.getQueueName().equalsIgnoreCase("clusterSingleQueue1"), "Queue created in MB node 1 not exist");
```

Comment:
I assume the purpose of creating several queues at "init" method was to fix the assert mentioned here. The actual issue was that the consumer was not create prior to this assert. 
My suggested fix is : Remove the queue creation in "init" method. Then move line 118 and 119  before this assert(prefer to line 106). That should fix it. This issue is there in another test case,  I am currently fixing it.